wolfssl/wolfcrypt/settings.h

```diff
@@ -2685,6 +2685,14 @@ extern void uITRON4_free(void *p) ;
     #define WOLFSSL_IPV6
 #endif
 
+/* ---------------------------------------------------------------------------
+ * ASN Library Selection (default to ASN_TEMPLATE)
+ * ---------------------------------------------------------------------------
+ */
+#if !defined(WOLFSSL_ASN_TEMPLATE) && !defined(WOLFSSL_ASN_ORIGINAL) && \
+    !defined(NO_ASN)
+    #define WOLFSSL_ASN_TEMPLATE
```

Comment:
We need to clearly document this in the ChangeLog.md. For people using user_settings.h this should not cause issues, but if it does let's make sure its known.

Reply:
Now or on the next release?

Reply:
Please start a new entry in the ChangeLog.md for the next now:

```
# wolfSSL Release X.Y.Z (TBD)

NOTE: This release switches the default ASN.1 parser to the new ASN template code. If the original ASN.1 code is preferred define `WOLFSSL_ASN_ORIGINAL` to use it. See PR #7199.
```

Reply:
Done